gm4_orbis/data/gm4_orbis/functions/structure/spawn.mcfunction

```diff
@@ -6,8 +6,9 @@ function #gm4_orbis:structure/post_spawn
 scoreboard players add structure_count gm4_orbis_config 1
 
 kill @e[tag=gm4_orbis_structure_pos]
-summon area_effect_cloud ~ ~ ~ {Tags:[gm4_orbis_structure_pos]}
+summon marker ~ ~ ~ {CustomName:'"gm4_orbis_structure_pos"',Tags:[gm4_orbis_structure_pos]}
```

Comment:
No point in naming the entity when it only exist during this function

Reply:
Naming all entities is good practice in case the entity stays around longer than it should. It makes it easier to debug when you can just /say @e and discover where the problematic entities are from.

It shouldn't be any slower since there's already nbt being set